mllib/src/main/scala/org/apache/spark/ml/param/shared/sharedParams.scala

```diff
@@ -234,6 +234,21 @@ private[ml] trait HasFitIntercept extends Params {
 }
 
 /**
+ * (private[ml]) Trait for shared param skipInvalid.
+ */
+private[ml] trait HasSkipInvalid extends Params {
```

Comment:
These changes should be implemented in `SharedParamsCodeGen.scala`.

See L23 of this file `// DO NOT MODIFY THIS FILE! It was generated by SharedParamsCodeGen.`